### ISSUE TITLE
Opening obsidian sometimes result in image not loading 

### ISSUE THREAD

**Issue by SKnight1207** [zsviczian/obsidian-excalidraw-plugin#552]
**Describe the bug** **Expected behavior** **Screenshots**

**To Reproduce**
1. Close obsidian after pasting some images in ACTIVE PANE
2. Open obsidan and that PANE again
3. See error

![closing obs](https://user-images.githubusercontent.com/63585568/160265246-c445c59c-952d-4f1d-bda3-3bab240a1c58.png)

**Environment (please complete the following information):**
 - OS including version: Windows 10 on PC
 - Plugin version: Excalidraw 1.6.20 (latest)
 - Obsidian version: 0.13.31 for Windows (latest)

**Additional context**

**Comment by zsviczian** [comment]
Yes, I am aware of this issue and looking for the root cause, but haven't found it yet.
When this happens you can switch to another document and back, and the images typically will load without issues. 

I found this seems to happen when my PC is busy with some other tasks (e.g. rendering a video in the background or Obsidian is reindexing files, etc.). That said I am actively searching for the root cause for a few weeks now, and making changes to limit the ferquency of this happening.
